packages/wrangler/src/__tests__/configuration.test.ts

```diff
@@ -644,6 +645,13 @@ describe("normalizeAndValidateConfig()", () => {
             preview_bucket_name: "R2_PREVIEW_2",
           },
         ],
+        services: [
+          {
+            binding: "SERVICE_BINDING_1",
+            service: "SERVICE_BINDING_1",
```

Comment:
NIT: I would give the service a different name to its binding, just to be sure we haven't accidentally switched them.

```suggestion
            service: "SERVICE_TYPE_1",
```